prow/plugins/releasenote/releasenote.go

```diff
@@ -219,6 +219,11 @@ func handlePullRequest(pc plugins.Agent, pr github.PullRequestEvent) error {
 }
 
 func shouldHandlePR(pr *github.PullRequestEvent) bool {
+	//Do not handle the PR when it's merged
```

Comment:
Actually, I was certain this is the case, but after a second look round, not so sure. Let me think about this before further discussion